ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetTransactionCountTest.java

```diff
@@ -71,6 +71,21 @@ public void verifyTransactionCount() {
         (JsonRpcSuccessResponse) privGetTransactionCount.response(request);
 
     assertThat(response.getResult()).isEqualTo(String.format("0x%X", NONCE));
-    verify(privacyController).determineNonce(senderAddress, privacyGroupId, ENCLAVE_PUBLIC_KEY);
+    verify(privateNonceProvider).getNonce(senderAddress, PRIVACY_GROUP_ID);
   }
+
+  //  @Test
+  //  public void returnPrivacyDisabledErrorWhenPrivacyIsDisabled() {
```

Comment:
Why is this test commented out?

Reply:
Looks like the test is the wrong place... The endpoint will not be available if privacy is disabled.
I have removed it.